_docs/xcm/hrmp-channel.md

```diff
@@ -133,9 +133,26 @@ tokenDecimals: 10
 ```
 
 ## Turing Staging
-Parachain sovereign account: 5Ec4AhNzVAVwDR54oqgWQ5YSoPbxXathSrQqTmqkF5o7xe6y
+The below details can be viewed on [Rococo Subscan](https://rococo.subscan.io/parachain/2114).
+
+Sovereign Account (Rococo V1): `5Ec4AhNzVAVwDR54oqgWQ5YSoPbxXathSrQqTmqkF5o7xe6y`
+
+Sovereign Account (Turing Staging): `5Dt6dpkWPwLaH4BBCKJwjiWrFVAGyYk3tLUabvyn4v7KtESG`
+
+Sovereign Account (Parachain): `0x7369626c42080000000000000000000000000000000000000000000000000000`
 
 ## Turing Network
-Encoded account: `0x7061726142080000000000000000000000000000000000000000000000000000`\
-[Turing open channel call](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing.oak.tech#/extrinsics/decode/0x29000101000214000400000000070088526a741300000000070088526a7400060003005ed0b2183c01dc070000140d010004000101007061726142080000000000000000000000000000000000000000000000000000) - do this call from sudo tab or via governance.
+The below details can be viewed on [Kusama Subscan](https://kusama.subscan.io/parachain/2114).
+
+Sovereign Account (Kusama): `F7fq1is7XWry4tWaYVZJ2uSwytCLFhsuEFarS7hit1chXtq`
+
+Sovereign Account (Turing Network): `67y2xkd8rZD7gapMM7EA8pug4rSvUCQjaoTm95mVEcTiR3w1`
```

Comment:
Does this line mean the Sovereign account on Turing?

It should be `68gzVdFcwnNUcwiExdbioBwGcktc2EZP9KQ1zvdTQn9WVdmn`.